src/request/reducers/RequestDetailRequestReducer.ts

```diff
@@ -150,7 +151,52 @@ const urlReducer = createRequestReducer<string>(
         return request ? request.url : ''
     });
 
-const webRequestReducer = createRequestReducer<{ body: string, formData: { [key: string]: string }, headers: { [key: string]: string } }>(
+function getValueAtKeyCaseInsensitive<T>(values: { [key: string]: T }, searchKey: string): T {
+    const normalizedSearchKey = searchKey.toLowerCase();
+    let foundValue = undefined;
+
+
+    _.forEach(values, (value, key) => {
+        if (key.toLowerCase() === normalizedSearchKey) {
+            foundValue = value;
+
+            return false;
+        }
+    });
+
+    return foundValue;
+}
+
+function getMediaType(contentType: string): string {
+    const index = contentType.indexOf(';');
+
+    if (index >= 0) {
+        return contentType.slice(0, index);
+    }
+    else {
+        return contentType;
+    }
+}
+
+function getFormData(request: IWebRequestPayload): { [key: string]: string } {
+    if (request && request.body && request.body.form && !_.isEmpty(request.body.form)) {
+        return request.body.form;
+    }
+
+    if (request && request.body && request.body.content && request.headers) {
+        const mediaType = getMediaType(getValueAtKeyCaseInsensitive(request.headers, 'content-type'));
+
+        if (mediaType.toLowerCase() === 'application/x-www-form-urlencoded') {
+            const formData = querystring.parse(request.body.content);
+
+            return formData;
+        }
+    }
```

Comment:
Yep, I had intended to do exactly that last night (in the morning), but forgot.  Thanks for catching it.